liquid_node/configuration.py

```diff
@@ -486,16 +487,25 @@ def is_app_enabled(self, app_name):
 
     @classmethod
     def _validate_collection_name(self, name):
-        if not name.islower():
+
+        # collection names are restricted by minio and elasticsearch
+        # for reference see:
+        # https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-create-index.html#indices-create-api-path-params
+        # https://docs.abp.io/en/abp/latest/Blob-Storing-Minio#options
+
+        search_forbidden_char = re.compile(r'[^a-z0-9-]').search
```

Comment:
Wait -- uppercase is OK, right? underscores? dots?

In prod, we have loads of uppercase collection names with dashes. We don't have any dots or underscores.

Reply:
I didn't get the correct docs there. It should be the S3 bucket rules that apply? (https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html).
Still underscores and capital letters seem to be not allowed. Dots are allowed but then we would have to handle some special cases so maybe we refrain from allowing them at all.

Reply:
Here's our confusion described here:
https://github.com/minio/minio/issues/9088#issuecomment-595227401

> Existing directories on your disk in standalone mode are already supported to have _ or capitalized names.
> We do not support creating new buckets in this manner because of AWS S3 DNS requirements.

They accept the capitalized names when they already exist on disk. We have some of those.

If you add the restriction here, the service will refuse to deploy the already existing collections.

We don't want any special logic here relatetd to old vs. new collections.

We also don't want to rename all of the collections into lowercase, because that will break the existing links for the old collections, and we also don't have any kind of collection aliases or redirects.

Things to do:
- add uppercase to the check
-